ts/components/session/SessionClosableOverlay.tsx

```diff
@@ -266,3 +290,30 @@ export class SessionClosableOverlay extends React.Component<Props, State> {
     }
   }
 }
+
+const MessageRequestList = () => {
+  // get all conversations with (accepted / known)
+  const lists = useSelector(getLeftPaneLists);
+  const unapprovedConversations = lists?.conversations.filter(c => {
+    return !c.isApproved;
+  }) as Array<ConversationListItemProps>;
+  console.warn({ unapprovedConversationsListConstructor: unapprovedConversations });
+  return (
+    <div className="message-request-list__container">
+      {unapprovedConversations.map(conversation => {
+        return <MessageRequestListItem conversation={conversation} />;
+      })}
+    </div>
+  );
+};
+
+// const MessageRequestListItem = (props: { conversation: ConversationModel }) => {
+const MessageRequestListItem = (props: { conversation: ConversationListItemProps }) => {
+  const { conversation } = props;
+  return (
+    <MemoConversationListItemWithDetails
+      isMessageRequest={true}
+      {...conversation}
+    ></MemoConversationListItemWithDetails>
```

Comment:
probably a good idea to write a selector again or a custom hook. It me up for instructions